packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -70,10 +70,34 @@ export class MeetWidget extends Component {
     props.hangupCall(media.get(`call`));
   }
 
+  @autobind
+  handleStartSendingAudio() {
+    this.props.media.get(`call`).startSendingAudio();
```

Comment:
Yea... I went back and forth on this one. Originally I was having redux thunks do this, but if we use the event listeners then a double event fire would happen. We unfortunately can't get rid of the events because really the `call` object is the single source of truth.